from_dict/_from_dict.py

```diff
@@ -63,76 +62,86 @@ def __eq__(self, o: object) -> bool:
 
 
 if IS_GE_PYTHON38:
-
-    def get_origin(t):
-        return typing.get_origin(t)
-
-    def get_args(t):
-        return typing.get_args(t)
-
+    from typing import get_origin, get_args
```

Comment:
Great improvement! Would you mind separating these improvements, which do not directly have to do with the change, into their own commit? That would reviewing the change a lot easier :).

Reply:
Yes, I can do that. I was thinking about that after I started the pull request.

Reply:
Okay, it is done. Hopefully I did not split them out into too may pull requests for you.
Besides the error location one, the others done have any changes that are client visible. 

After the other pull requests have been merged I will merge with main again and the PR should only show the core changes.

Reply:
@wchresta, I split out some of the changes into other pull requests. Are there other changes you want split out?